lottie-compose/src/main/java/com/airbnb/lottie/compose/LottieAnimation.kt

```diff
@@ -88,6 +90,7 @@ fun LottieAnimation(
     clipTextToBoundingBox: Boolean = false,
     fontMap: Map<String, Typeface>? = null,
     asyncUpdates: AsyncUpdates = AsyncUpdates.AUTOMATIC,
+    isSafeMode: Boolean = false,
```

Comment:
```suggestion
    safeMode: Boolean = false,
```

Let's remove `is` to be consistent with the other boolean names here.

Reply:
@gpeal done